CONTRIBUTING.md

```diff
@@ -65,6 +65,7 @@ Run the dev build task for your browser from the 'Build' section above. The gene
 After running the build task it will continue watching for changes to any of the source files. After saving any changes to these files it will automatically rebuild the `dev` directory for you.
 
 ### Testing
+- Install the required test submodules: `git submodule update --init --recursive`
```

Comment:
This is already done in the postinstall step, does this need to also be explicitly run in the testing instructions?

Reply:
I added the postinstall after the above, but its probably still worth calling out if you have an existing install